widget/list.go

```diff
@@ -95,6 +95,19 @@ func (l *List) MinSize() fyne.Size {
 	return l.BaseWidget.MinSize()
 }
 
+func (l *List) moveFocusTo(id ListItemID) {
+	if l.scroller == nil {
+		return
+	}
+	y := (float32(id) * l.itemMin.Height) + (float32(id) * theme.SeparatorThicknessSize())
```

Comment:
I know that it was like this before, but this can be simplified. You can rewrite it as the following:
```suggestion
	y := float32(id) * (l.itemMin.Height) + theme.SeparatorThicknessSize())
```

Reply:
Out of interest, I double checked to see if the resulting assembly were different or if the compiler would be smart enough to optimize it away. It does look like this change indeed does save us a multiplication, as I expected 🙂 https://go.godbolt.org/z/1556hfGnr

Reply:
That's a great catch! Thanks for the suggestion!

Reply:
You're welcome. Feels a bit strange though that the compiler wouldn't be able to optimize it away. Opened an issue over at golang/go, https://github.com/golang/go/issues/45760.

Reply:
Comments in that issue mentioned that rounding will make this create different values. https://play.golang.org/p/ziUoFO0OhNT

I think you might want to review this change. Sorry 😬

Reply:
Oh, sorry looks like there was a typo in that message. I meant that you might need to revert the change. It can cause issues with rounding and be less precise that what it was before.

Reply:
Thanks for reverting it back to how it was. You can remove the parenthesis if you like, but now exactly needed.

Reply:
I prefer parenthesis from a code readability perspective. Can remove them if it goes against the project design guidelines.

Reply:
I don’t think it goes against any guidelines, no. I just feel that they are unnecessary given that order of operation means that they have no meaning. This is not really hard to read code in the first place, but if you want to keep them, feel free to do so. I’m not going to stop you 🙂